src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -111,14 +111,14 @@ export default () => `
 				<div class="resources-container">
 					<h2>${escape(localize('welcomePage.resources', "Resources"))}</h2>
 					<div class="tabs">
-						<input class="input" name="tabs" type="radio" id="tab-1" checked="checked" />
+						<input tabindex="-1" class="input" name="tabs" type="radio" id="tab-1" checked="checked" />
 						<label class="label" for="tab-1" tabIndex="0">${escape(localize('welcomePage.history', "History"))}</label>
 						<div class="panel">
 							<div class="recent history">
 								<div class="flex list-header-container">
 									<i class="icon-document themed-icon"></i>
-									<h4 class="list-header">${escape(localize('welcomePage.name', "Name"))}</h4>
-									<h4 class="list-header-last-opened">${escape(localize('welcomePage.lastOpened', "Last Opened"))}</h4>
+									<label tabindex="0" class="list-header"><b>${escape(localize('welcomePage.name', "Name"))}</b></label>
```

Comment:
Is there a reason you're using label instead of something like span? Label is usually used when you're labeling a specific element such as a checkbox right?

Reply:
I have it skipping the checkbox for now because it is an unused feature that hasnt been built in yet. The future feature is to allow users to tab between history and pinned files. The checkbox provides no use for the user right now. But the styling and markup are built around what the new feature will be utilizing.

Reply:
https://www.w3schools.com/tags/tag_label.asp, i agree with Charles here. @v-bbrady could you please make it span?